integration_tests/test_get_requests.py

```diff
@@ -43,6 +42,19 @@ def check_response(r: Response):
     check_response(get(f"/{function_type}/template"))
 
 
+@pytest.mark.benchmark
+@pytest.mark.parametrize("function_type", ["sync", "async"])
+def test_url_for(function_type: str, session):
+    def check_response(r: Response):
+        assert r.status_code == 200
+        assert r.text == f"<a href='/{function_type}/url_for'>/{function_type}/url_for</a>"
```

Comment:
this is not a test for `url_for` , this is testing whether `sync` or `async` is present in the test.